src/blog/2022/08/flowforge-08-released.md

```diff
@@ -0,0 +1,71 @@
+---
+title: FlowForge 0.8 released
+subtitle: Inter-Project Communication, Default Teams, and realtime device management.
+description: Inter-Project Communication, Default Teams, and realtime device management.
+date: 2022-08-04 12:00:00.0
+authors: ["sam-machin"]
+video: BBNzFInc_2E
+---
+
```

Comment:
Special thanks to: 
```
@HaroldPetersInskipp made their first contribution in https://github.com/flowforge/flowforge/pull/812
@Steveorevo made their first contribution in https://github.com/flowforge/flowforge/pull/818
```

Reply:
@ZJvandeWeg was that meant to be an edit?

Reply:
No, those lines are from: https://github.com/flowforge/flowforge/releases/tag/v0.8.0
But think we should thank them somewhere in the blog

Reply:
added a contributors section